company-kata/company-kata.iml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module external.linked.project.id=":company-kata" org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8" inherit-compiler-output="false">
```

Comment:
Good question.  I think it is something the latest version of IntelliJ has added but honestly I am not sure what it is for.  I will do some research to see if there is anything I should change here.